CAIPs/caip-25.md

```diff
@@ -108,6 +134,8 @@ Each object should be keyed to the scope of a `sessionScopes` member to which it
 All properties of each object in `scopedProperties` MUST be interpreted by the respondent as proposals or declarations rather than as requirements.
 In addition to making additional properties of or metadata about the corresponding `sessionScopes` member explicit, they can also annotate, support, or extend the negotiation of scope proposals (e.g., providing connection information about unfamiliar scopes, or which accounts to expose to each).
 
+When there is an active session between the wallet and the caller for which the wallet assigned a `sessionId` in it's initial response, that same `sessionId` should be added to the `wallet_createSession` request to update the associated session.
+
```

Comment:
```suggestion
```
I'd say we move this down 2 paragraphs, it seems to have gotten stuck here as an artefact of editing (probably by me or merge conflicts)